anesthetic/samples.py

```diff
@@ -573,7 +573,7 @@ def remove_burn_in(self, burn_in, reset_index=False, inplace=False):
             data = data.reset_index(drop=True, inplace=inplace)
         return data
 
-    def Gelman_Rubin(self, params=None):
+    def Gelman_Rubin(self, params=None, return_per_param=False):
```

Comment:
Perhaps `per_param` instead of `return_per_param`?

Reply:
Have been contemplating that, too, you suggesting it settles it.

Done in https://github.com/handley-lab/anesthetic/pull/345/commits/299fddb31541ac8aa7e6ca72407b32991661cc6b.